### PR TITLE
fix: add versioning for bump-version to succeed

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments

In the prior commit, 1da9c7c2b9b75ad19f62f441f72761ea6fd38cb7, I had a single-line oversight of adding `versioning = "CalVer"` to `setup.cfg`, which caused some downstream issues.  In addition to fixing that here, we made it more obvious for the future via https://github.com/SeisoLLC/goat/pull/163 and https://github.com/SeisoLLC/cookiecutter-python/pull/97

## Pull Request Checklist

Thank you for submitting a contribution to `easy_infra`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:

- [x] Rebase your branch against the latest commit of the target branch
- [x] If you are adding a dependency, please explain how it was chosen
- [x] If manual testing is needed in order to validate the changes, provide a testing plan and the expected results
- [x] If there is an issue associated with your Pull Request, link the issue to the PR.

### CODE REVIEW

File: Taskfile.yml [hunk 1]
```diff
@@ -139,9 +139,10 @@ tasks:
...
     cmds:
       - task: py:update
-      - pipenv run python -c \
-        'from distutils.util import strtobool; \
-        debug = bool(strtobool("{{.DEBUG}}")); \
+      - |
+        pipenv run python -c \
+        'from distutils.util import strtobool;
+        debug = bool(strtobool("{{.DEBUG}}"));
         from {{.PROJECT_SLUG}} import utils; utils.update(debug=debug)'
```
Comment (ChiefHolland) [inline Taskfile.yml:146]: Please? For OCD's sake?

```suggestion
from {{.PROJECT_SLUG}} import utils;
utils.update(debug=debug)'
```

**Comment by JonZeolla** [comment]
@ChiefHolland you got it CC @seanctech

**Review by ChiefHolland — APPROVED** [review]
LGTM!

**Comment by JonZeolla** [comment]
Pending https://github.com/SeisoLLC/goat/pull/164

**Review by seanctech — APPROVED** [review]
LGTM 🚀

**Pull request merged**